### PR TITLE
feat: new tab and component for pictograms

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Adding a new component to present and download pictograms hosted on AWS.

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#1113](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1113)

**How to test**  
Not sure how to test this done due to local cms database changes that are not yet in productions.
Let's sync when testing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "jasmine-core": "~3.4.0",
     "jasmine-spec-reporter": "~4.2.1",
     "jszip": "^3.5.0",
+    "jszip-utils": "^0.1.0",
```
Comment (theJohnnyMe) [inline package.json:53]: New package that enables zipping zipped files, from the same author as jszip...

File: src/app/components/page/tab-content/tab-content.component.html [hunk 2]
```diff
@@ -37,6 +37,11 @@
           <div class='sdds-icon-list' icon-list [icons]='section.Icons' [category]='section.category' [lastUpdate]='item.content.updated_at'></div>
         </ng-template>
 
+        <!-- Pictogram List -->
+        <ng-template [ngIf]="section.__typename === 'ComponentContentPluginPictogramList'">
```
Comment (theJohnnyMe) [inline src/app/components/page/tab-content/tab-content.component.html:41]: @gaonita  - this is how CMS and Website are connected. So basically "ComponentContentPluginPictogramList" is a component made in CMS and here website is importing it and loading content from it.

File: src/app/components/pictogram-list/pictogram-list.component.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                <span class="delete-from-list" (click)='removePictogram(selectedPictogram);'><sdds-icon name='scania-cross'></sdds-icon></span>
+              </li>
+            </ul>
+          </div>
+          <div class="col-md-6">
+            <p>No. of pictograms: <b>{{checkedPictograms.length}}</b></p>
+          </div>
+
+          <div class="col-md-6 text-right">
+            <button class="sdds-btn sdds-btn-primary" (click)='downloadSelectedAsZip()' [disabled]="checkedPictograms.length === 0">
```
Comment (theJohnnyMe) [inline src/app/components/pictogram-list/pictogram-list.component.html:88]: @gaonita  - How to write disabled state on the button and control it dynamic.

**Review by mmexvr — APPROVED** [review]
Looks good

**Pull request merged**